### ISSUE TITLE
Bug when accessing tender cargo

### ISSUE THREAD

**Issue by scaires** [scaires/CTS#47]
```
/Users/stevelopment/Workspace/CTS/index.js:872
		var _cargo = _shipcontact.cargo.containersForContractIds(data.player.journal.acceptedContractIds());
		                         ^

TypeError: Cannot read property 'cargo' of undefined
    at menuCargo (/Users/stevelopment/Workspace/CTS/index.js:872:28)
    at Object.<anonymous> (/Users/stevelopment/Workspace/CTS/index.js:1313:4)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:149:9)
```

**Comment by scaires** [comment]
Looks like `isDocked` is true, but because it's a tender, there's no docked ship. Could require reworking tender stuff or just fixing this bug.

**Comment by scaires** [comment]
Fixed in 1defe85466da55c8b47d5bcbb3da286af459f808
